src/utils/router/NotLoggedInRoute.js

```diff
@@ -15,7 +17,27 @@ export function NotLoggedInRoute({ children, ...rest }) {
         if (user && user.displayName) {
           return <Redirect to={from || routes.lobby()} />;
         } else {
-          return children;
+          return (
+            <>
+              <Helmet>
+                {gameName && <title>Join to play {gameName} | Corona Games</title>}
+                <meta property="og:title" content="Play with your friends!" />
+                {gameName && gameID && (
+                  <meta
+                    property="og:description"
+                    content={`Click this link to play ${gameName} with your friends. They wait for you in room #${gameID}`}
```

Comment:
We could probably fetch some room info here to show number of players or something like this, but do we really need it now?